generic/secrets/security/detected-github-token.yaml

```diff
@@ -1,8 +1,18 @@
 rules:
   - id: detected-github-token
-    pattern-regex: |-
-      gh[pousr]_[A-Za-z0-9_]{36,251}
-    languages: [regex]
+    patterns:
+    - pattern-either:
+      - pattern: |
+          $VAR = $SECRET
+      - pattern: |
+          $VAR: $SECRET
```

Comment:
The secret could also be in a URL somewhere.
If we try the old and new versions of this rule at scale, what do we find?

Reply:
I'll add in variations for string and url-finding for this one!